lnwallet/btcwallet/btcwallet_test.go

```diff
@@ -0,0 +1,168 @@
+package btcwallet
```

Comment:
nit: this change should probably be added to the commit that added the function.